backtracking/graph_coloring.cpp

```diff
@@ -1,60 +1,89 @@
-#include <stdio.h>
+/**
+ * @file
+ * @brief brief one line description here
+ */
+#include <iostream>
 
-// Number of vertices in the graph
-#define V 4
-
-void printSolution(int color[]);
+// created a namespace to ensure all globals and functions are
```

Comment:
please refer [this line](https://github.com/TheAlgorithms/C-Plus-Plus/blob/076d156302cc93101eb56958c1cbd9d5829a4251/sorting/shell_sort2.cpp#L36) for reference on how to document a namespace. Thanks.